lib/vagrant-service-manager/command.rb

```diff
@@ -248,6 +248,13 @@ def perform_service(operation, service)
             PluginUtil.generate_kubeconfig(machine, @env.ui, File.dirname(ServiceManager.bin_dir))
           end
         end
+      rescue StandardError => e
+        if operation == 'restart' && e.message.include?("Start #{service} first")
+          @env.ui.error I18n.t('servicemanager.commands.operation.start_service_first')
+          return
```

Comment:
@navidshaikh Since, sccli is not offering that feature, we are forced to come up with messaging at present.